docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
+tar balls on the pbench server.
 
---sysinfo STR,[STR]  
-comma-separated values of system information to be collected; available: default, none, all, ara, block, insights, kernel_config, libvirt, security_mitigations, sos, stockpile, topology
+`--show-server`  
+This will not move any results, but resolve and
+then display the pbench server destination for results.
 
---pbench-pre STR  
-Path to the script which will be executed before tools are started  
-_NOTE: --pbench-pre is not compatible with --use-tool-triggers_
+`--xz-single-threaded`  
+This will force the use of a single
+thread for locally compressing the result tar balls.
 
---pbench-post STR  
-Path to the script which will be executed after tools are stopped and postprocessing is complete  
-_NOTE: --pbench-post is not compatible with --use-tool-triggers_
+`--help`  
+Show this message and exit.
 
---use-tool-triggers  
-Use tool triggers instead of normal start/stop around script;  
-_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+---
 
---no-stderr-capture  
-Do not capture the stderr of the script to the result.txt file
+##### pbench-copy-results
+
+---
+
+**NAME**  
+`pbench-copy-results`, Copies result tarball to the 0.69 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]`
+
+**DESCRIPTION**
+Push the benchmark result to the Pbench Server without removing it from the local host .We use id_rsa when pushing to a 0.69 Pbench Server.
 
---help  
+**OPTIONS**  
+`--user`  
+This option value is required if not provided by the
+'PBENCH_USER' environment variable; otherwise, a value provided
+on the command line will override any value provided by the
+environment.
+
+`--controller`  
+This option may be used to override the value
+provided by the 'PBENCH_CONTROLLER' environment variable; if
+neither value is available, the result of 'hostname -f' is used.
+(If no value is available, the command will exit with an error.)
+
+`--prefix`  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+files on the 0.69 Pbench Server.
+
+`--show-server`  
+This will not move any results, but resolve and
+then display the pbench server destination for results.
+
+`--xz-single-threaded`  
+This will force the use of a single
+thread for locally compressing the result tar balls.
+
+`--help`  
 Show this message and exit.
+
+---
+
+##### pbench-results-move
+
+---
+
+**NAME**  
+`pbench-results-move`, Move results tarball to the server to a 1.0 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]`
+
+**DESCRIPTION**  
+Move result directories to the configured Pbench server. We use an API Key authentication token when pushing results to a 1.0 Pbench Server
```

Comment:
As I mentioned above, the use of `We` here should probably be reworked.  Also, "use an API Key authentication" is pretty weak (and the "when" implies a conditionality which is not actually present).  The fact that the `--token` option is required already makes that fairly clear, so we should be saying something else, here.

> This command uploads one or more result directories to the configured v1.0 Pbench Server.  The specified API Key is used to authenticate the user and to establish ownership of the data on the server.  Once the upload is complete, the result directories are, by default, removed from the local system.